pkgs/applications/misc/ringing-lib/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub, lib, autoreconfHook, pkg-config, readline, xercesc }:
+stdenv.mkDerivation rec {
+  pname = "ringing-lib";
+  version = "0.4.0-pre";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "cce750629703814a88030cd342369fc491dafbd6";
+    sha256 = "sha256-jbX61b/8zwBfxkv3P3OQwh295iu+9rNdJMazR2lMdHw=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs = [ readline xercesc ];
+
+  meta = with lib; {
+    description = "Library of C++ classes and utilities for change ringing";
+    homepage = "https://ringing-lib.github.io/";
+    license = with licenses; [ gpl2Plus lgpl2Plus ];
+    maintainers = with maintainers; [ jshholland ];
+  };
```

```suggestion
    platforms = platforms.linux;
  };
```
Doesn't build successfully on MacOS.